test/compositemark/errorbar.test.ts

```diff
@@ -1121,4 +1272,61 @@ describe('normalizeErrorBar with aggregated error input', () => {
       );
     }).toThrow();
   });
+
+  it('should produce a correct tooltip title for errorbar with pre-aggregated error value', () => {
+    const outputSpec = normalize(
+      {
+        data: {url: 'data/population.json'},
+        mark: {type: 'errorbar', extent: 'stderr'},
+        encoding: {
+          x: {field: 'age', type: 'ordinal'},
+          y: {field: 'people', type: 'quantitative'},
+          yError: {field: 'people_error', type: 'quantitative'}
+        }
+      },
+      defaultConfig
+    );
+
+    const layer = isLayerSpec(outputSpec) && outputSpec.layer;
+    expect(layer).toBeTruthy();
+    for (const unitSpec of layer) {
+      const encoding = isUnitSpec(unitSpec) && unitSpec.encoding;
+      expect(encoding).toBeTruthy();
+      const tooltip = encoding.tooltip;
+      expect(tooltip).toEqual([
+        {field: 'upper_people', title: 'people + people_error', type: 'quantitative'},
+        {field: 'lower_people', title: 'people - people_error', type: 'quantitative'},
+        {field: 'age', type: 'ordinal'}
+      ]);
+    }
+  });
+
+  it('should produce a correct tooltip title for errorbar with pre-aggregated error value', () => {
+    const outputSpec = normalize(
+      {
+        data: {url: 'data/population.json'},
+        mark: {type: 'errorbar', extent: 'stderr'},
+        encoding: {
+          x: {field: 'age', type: 'ordinal'},
+          y: {field: 'people', type: 'quantitative'},
+          yError: {field: 'people_error1', type: 'quantitative'},
+          yError2: {field: 'people_error2', type: 'quantitative'}
+        }
+      },
+      defaultConfig
+    );
+
+    const layer = isLayerSpec(outputSpec) && outputSpec.layer;
+    expect(layer).toBeTruthy();
+    for (const unitSpec of layer) {
+      const encoding = isUnitSpec(unitSpec) && unitSpec.encoding;
+      expect(encoding).toBeTruthy();
+      const tooltip = encoding.tooltip;
+      expect(tooltip).toEqual([
+        {field: 'upper_people', title: 'people + people_error1', type: 'quantitative'},
```

Comment:
Why is there no tooltip for `y` if `y`, `yError`, and `yError2` are all provided?

Also, why is extent specified when users directly specify yError/yError2?